lib/pulse/core/PulseCore.Atomic.fsti

```diff
@@ -401,37 +401,39 @@ val ghost_gather
     (ghost_pts_to r v0 ** ghost_pts_to r v1)
     (fun _ -> ghost_pts_to r (op pcm v0 v1))
 
-val ghost_witnessed
-    (#a:Type u#1)
-    (#p:pcm a)
-    (r:ghost_ref p)
-    (f:property a)
-: Type0
-
-val ghost_witness
-    (#a:Type)
-    (#pcm:pcm a)
-    (r:ghost_ref pcm)
-    (fact:stable_property pcm)
-    (v:Ghost.erased a)
-    (pf:squash (forall z. compatible pcm v z ==> fact z))
-: stt_atomic
-    (ghost_witnessed r fact)
-    #Unobservable emp_inames
-    (ghost_pts_to r v)
-    (fun _ -> ghost_pts_to r v)
-
-val ghost_recall
-    (#a:Type u#1)
-    (#pcm:pcm a)
-    (#fact:property a)
-    (r:ghost_ref pcm)
-    (v:Ghost.erased a)
-    (w:ghost_witnessed r fact)
-: stt_atomic (v1:Ghost.erased a{compatible pcm v v1})
-    #Unobservable emp_inames
-    (ghost_pts_to r v)
-    (fun v1 -> ghost_pts_to r v ** pure (fact v1))
+// Unused?

```

Comment:
These are unused, but @mtzguido tells me that they are expected to be required soon. I will restore them in a follow-up PR.